website/server/libs/api-v3/buildManifest.js

```diff
@@ -39,23 +39,33 @@ export function getBuildUrl (url) {
   return `/${buildFiles[url] || url}`;
 }
 
-export function getManifestFiles (page) {
+export function getManifestFiles (page, type) {
   let files = manifestFiles[page];
 
   if (!files) throw new Error(`Page "${page}" not found!`);
 
   let htmlCode = '';
 
   if (IS_PROD) {
-    htmlCode += `<link rel="stylesheet" type="text/css" href="${getBuildUrl(page + '.css')}">`; // eslint-disable-line prefer-template
-    htmlCode += `<script type="text/javascript" src="${getBuildUrl(page + '.js')}"></script>`; // eslint-disable-line prefer-template
+    if (type !== 'js') {
```

Comment:
because otherwise passing no `type` would wouldn't work as neither js nor css files would be added